docs/installation.rst

```diff
@@ -22,6 +22,13 @@ Setting up Vagrant
   .. code-block:: bash
 
      >vagrant up
+  
+  Note: If you encouter error showing "Your machine doesn't supported NFS, or there is not an
+  adapter to enable NFS on this machine for Vagrant",then install ``nfs-kernel-server`` using the command:
```

Comment:
If we add these and other messages (see below) I wonder if we might want to present the error in a code block as well. I think the error above got a bit altered during writing, eg. *'doesn't supported'* above. Here's a link to all the errors (I think) for reference. I defer to @edmorley how best to present them.
https://github.com/edx/edx-platform/wiki/Simplified-install-with-vagrant